README.md

```diff
@@ -21,9 +21,9 @@ Currently, wsdoc is available in source format only. To install, you'll need mvn
 
 Once you've done this, the wsdoc jar will be available in your local Maven repository, probably at ~/.m2/repository/org/versly/versly-wsdoc/1.1-SNAPSHOT/versly-wsdoc-1.1-SNAPSHOT.jar
 
-Note, the Java 1.8 requirement is necessary to accommodate instructions to the annotation processor to recognize Java types introduced in 1.8.  If you intend to use wsdoc to document code that does not utilize newer types, a one-line change in AnnotationProcessor.java can be made.  Specifically, locate this line in AnnotationProcessor.java:
+Note, the Java 1.8 requirement is necessary to accommodate instructions to the annotation processor to recognize Java types introduced in 1.8. The current supported source version is Java 11. If you intend to use wsdoc to document code that does not utilize newer types, a one-line change in AnnotationProcessor.java can be made.  Specifically, locate this line in AnnotationProcessor.java:
```

Comment:
Should the above also indicate that Java 11 is now required?

Reply:
I was debating this. Looking at the definition of [supported source version](https://docs.oracle.com/en/java/javase/11/docs/api/java.compiler/javax/annotation/processing/SupportedSourceVersion.html), it states "..used to indicate the latest source version an annotation processor supports...". I interpreted that as a maximum, rather than a minimum. From that logic, it is very possible that not all java 11 annotations are supported, and my commit to readme is an exaggeration.  I could very well be wrong on this and happy to change it to java 11 required.